src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -33,36 +33,36 @@ <h5 *ngIf="workItem.id" class="detail-id pull-left" id="wi-detail-id">{{workItem
           <div class="col-md-8 col-sm-7 col-xs-12 wi-detail-title-blk">
             <div *ngIf="!headerEditable" (click)='openHeader()'
               id="title-click-div">
-                <h2 *ngIf="!loggedIn" id="wi-detail-title-ne" class="detail-title truncate">{{ workItem.attributes['system.title'] }}</h2>
-                <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{ workItem.attributes['system.title'] }}</h2>
-                <span *ngIf="loggedIn && workItem.id"
-                  class="pficon-edit marginL10 fl detail-title-edit-ico"
-                  id="workItemTitle_btn_edit"></span>
+              <h2 *ngIf="!loggedIn" id="wi-detail-title-ne" class="detail-title truncate">{{ workItem.attributes['system.title'] }}</h2>
+              <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{ workItem.attributes['system.title'] }}</h2>
+              <span *ngIf="loggedIn && workItem.id"
+                class="pficon-edit marginL10 fl detail-title-edit-ico"
+                id="workItemTitle_btn_edit"></span>
             </div>
             <div *ngIf="(loggedIn && headerEditable)" id="wi-title-div">
-                <div almEditable
+                <h2 almEditable
```

Comment:
There is a noticeable jump when we click on the Edit icon. We need to remove this jump.
![nonedit](https://cloud.githubusercontent.com/assets/1892722/22679432/3658c56c-ed28-11e6-9533-7a5af5c6c3ff.png)

![edit](https://cloud.githubusercontent.com/assets/1892722/22679439/3ed7641e-ed28-11e6-9ec6-2f2511221083.png)